snoop/archives.py

```diff
@@ -22,14 +22,14 @@
 else:
     CACHE_ROOT = None
 
-class MissingArchiveFile(Exception):
-    pass
+class MissingArchiveFile(models.BrokenDocument):
```

Comment:
You know, I just realised, this is confusing. `models.BrokenDocument` looks like it should be a table where we keep broken documents. Let's move it to a new `exceptions.py`.